### ISSUE TITLE
Frontend accessibility

### ISSUE THREAD

**Issue by tsherlockcraig** [interline-io/calact-network-analysis-tool#329]
My bad for not bringing this up before. My suspicion is that it's something that has naturally been done as the frontend has been designed, to the extent feasible, but I realized I should confirm, and then if needed we'll work appropriate changes in the the roadmap.

The frontend of the website should by WCAG 2.1 AA compliant. I'm guessing that for the filters selection and tabular interface, this is both reasonable and probably already mostly the case. The map interface is in my understanding necessarily visual, and as long as all information in the map interface is in the tabular interface, the tabular interface serves as the accessible alternative to the map interface.

@drewda let me know if you think anything above is inaccurate. We can discuss at a future Monday meeting the degree to which this requirement would necessitate significant development.

**Comment by drewda** [comment]
Yes, this has also been a concern at the back of my mind. Let's discuss on a future call, since I imagine @talorz-garnet and @NomeQ have relevant experience and pointers to share.

Also timely since Ian just did housekeeping to break out the UI widgets into their own public repo: https://github.com/interline-io/catenary We should make sure all of those have relevant accessibility attributes.

**Comment by NomeQ** [comment]
I've started doing a cursory review of the app. A lot of the fields have the basic attributes, but are missing things like correct group labels for checkbox groups, or focus and button behavior for menu items. I've started collecting the issues I find here: https://docs.google.com/spreadsheets/d/1PUxpUfGHk2RGSgo-aSG2sxIzLCk_eRK8Q0mGInucLXU/edit?usp=sharing

A few of these look like issues to correct with the component library (esp creating a parent component for accessible checkbox/radio button groups), but a number seem to be the with the app. We can talk about these at our next meeting and/or I can also help out in fixing some of these issues.

**Comment by drewda** [comment]
@NomeQ this is very helpful!

For component-level improvements, it would be helpful if you can create one issue per component in https://github.com/interline-io/catenary Feel free to take on any you'd like. Ian and I can take some on as well. It probably doesn't have to happen in any particular order.

For app-level improvements, those might be good to talk through in total and then decide how to divvy up into an order list.

**Comment by NomeQ** [comment]
@drewda I can do that, this was an incomplete list (mostly just keyboard tests), so I'll go ahead and do a complete review of the app and try to break up what is component vs app. During our Monday meeting we can set a time for a separate accessibility meeting.
